guides/v2.3/config-guide/cli/config-cli-subcommands-index.md

```diff
@@ -120,6 +120,20 @@ Product Price index has been rebuilt successfully in <time>
 Catalog Search index has been rebuilt successfully in <time>
 ```
 
+To reinstall triggers:
```

Comment:
Hi @konarshankar07. Thank you for providing the information. 
Currently, as a reader, I find this information a little bit confusing. I'm not sure what's a trigger, why I need to rebuild the triggers? So we a little bit out of context here. 

I would suggest extending the [following](https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-index.html#configure-indexers-1) section with a mention that
> The indexers-related database triggers are being added when the indexer mode is set to `schedule` and removed when the indexer mode is set to `realtime`.

Thank you!

Reply:
Hello @rogyar ,
I have updated the content as per your above feedback. My question is how we can reference this in MySQL section which is mention in the task?
Thanks